.github/pull_request_template.md

```diff
@@ -1,15 +1,52 @@
-Describe your changes here.
+_Authors: Please fill out this form carefully and completely._
 
-Closes # (type the issue number after # if applicable; otherwise remove this line)
+_Reviewers: By approving this Pull Request you are approving the code change, as well as its deployment and mitigation plans._
+_Please read this description carefully. If you feel there is anything unclear or missing, please ask for updates._
+
+#### List the issues that this change affects.
+<!--Every code change must address _at least 1_ issue. Fixes a bug, completes a task, every change
+      should have a corresponding issue listed here. If one does not already exist, create one. -->
+      
+Closes # (type the GitHub issue number after #)
+
+### What are you trying to accomplish?
+<!-- Provide a description of the changes. -->
```

Comment:
```suggestion
<!-- Provide the GitHub issue that this issue closes. Start typing the number or name of the issue after the # below -->

Closes #

<!-- Provide an overview of the changes, including any screenshots when helpful -->
```

Reply:
Resolved in commit https://github.com/primer/react/pull/3344/commits/6631c1b2ea7cfe9226201b606ef0261ba57ab1b8